psalm-baseline.xml

```diff
@@ -22,70 +22,21 @@
     </UndefinedDocblockClass>
   </file>
   <file src="Packages/Framework/Neos.Cache/Classes/Backend/MemcachedBackend.php">
-    <UndefinedClass occurrences="2">
-      <code>\Memcached</code>
```

Comment:
We run builds with memcached and redis extensions installed to test those backends, so the classes should not be undefined.